packages/react-kitchen-sink/src/react-query/FirestoreSnaphotManager.ts

```
@@ -1,15 +1,45 @@
-import { hashKey, type Query, type QueryCacheNotifyEvent, type QueryClient, type QueryKey } from '@tanstack/react-query'
+import {
```

ESLint

[error] Filename is not in kebab case. Rename it to `firestore-snaphot-manager.ts`.

(unicorn/filename-case)